### PR TITLE
Add support data for fabric monitoring in CONFIG_DB.

### PR DESCRIPTION
**Pull request by jfeng-arista**
Added support data for fabric monitoring in CONFIG_DB

The CONFIG_DB now has the FABRIC_MONITOR|FABRIC_MONITOR_DATA table for default value for fabric port monitoring. An example output of getting this table is: 

> sonic-db-cli CONFIG_DB hgetall "FABRIC_MONITOR|FABRIC_MONITOR_DATA"
{'monErrThreshCrcCells': '1', 'monErrThreshRxCells': '61035156', 'monPollThreshIsolation': '1', 'monPollThreshRecovery': '8'}

The CONFIG_DB now also has a table for each fabric port for its isolate status. 
An example output of getting this table is: 

> sonic-db-cli CONFIG_DB hgetall "FABRIC_PORT|Fabric20"
{'alias': 'Fabric20', 'isolateStatus': 'False', 'lanes': '20'}

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

**Comment by kenneth-arista** [comment]
@rlhui @arlakshm for viz

**Comment by rlhui** [comment]
@sanmalho-git  - please add reviewers as you see fit, thanks.

**Review by arlakshm — COMMENTED** [review]
add comments, Please add some unit tests for this.

File: device/arista/x86_64-arista_7800r3_48cq2_lc/fabric_port_config.ini [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# name              lanes     isolateStatus
```
[the remark this answers is not in the document]
Comment (jfeng-arista) [reply device/arista/x86_64-arista_7800r3_48cq2_lc/fabric_port_config.ini:1]: Different sku might have different # of fabric ports, so we added file in the in the sku level.

File: device/arista/x86_64-arista_7800r3_48cqm2_lc/fabric_port_config.ini [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# name              lanes     isolateStatus
```
[the remark this answers is not in the document]
Comment (jfeng-arista) [reply device/arista/x86_64-arista_7800r3_48cqm2_lc/fabric_port_config.ini:1]: Different sku might have different # of fabric ports, so we added file in the in the sku level.

[the remark this answers is not in the document]
Comment (arlakshm) [reply device/arista/x86_64-arista_7800r3_48cqm2_lc/fabric_port_config.ini:1]: The fabric port defined here is based on the ASIC irrespective of sku or hwsku? so should the files be defined in ASIC directory?

[the remark this answers is not in the document]
Comment (jfeng-arista) [reply device/arista/x86_64-arista_7800r3_48cqm2_lc/fabric_port_config.ini:1]: When we worked on this, we thought different sku might have some ports not designed to be connected, so we tried to follow the port_config.ini and listed them in the sku level to cover this case, also could be easy to extend to cover other things might added related to sku in the future.

**Comment by kenneth-arista** [comment]
@arlakshm looks like @jfeng-arista addressed all comments. Can we get this merged before the 202305 cut-off?

File: src/sonic-config-engine/portconfig.py [hunk 3]
```diff
@@ -100,6 +100,89 @@ def get_hwsku_file_name(hwsku=None, platform=None):
             return candidate
     return None
 
+def get_fabric_monitor_config(hwsku=None, asic_name=None):
+    config_db = db_connect_configdb(asic_name)
+    if config_db is not None:
+       fabric_monitor_global = config_db.get_table("FABRIC_MONITOR")
+       if bool( fabric_monitor_global ):
+          return fabric_monitor_global
```
[the remark this answers is not in the document]
Comment (arlakshm) [reply src/sonic-config-engine/portconfig.py:108]: @jfeng-arista, @kenneth-arista  can you address this comment?

**Review by arlakshm — COMMENTED** [review]
LGTM

**Comment by StormLiangMS** [comment]
@gechiang  if this is a bug fix and we needed in 202305, pls have ADO to track.

**Comment by gechiang** [comment]
@StormLiangMS 
MSFT ADO: 24595606

Thanks!

**Pull request merged**